public/data/SiddhiBhanushali.json

```
@@ -0,0 +1,17 @@
+{
+  "name": "Siddhi Bhanushali",
+  "bio": "Web development & open source enthusiast",
```

It should be `Web developer`.
```suggestion
  "bio": "Web developer & open source enthusiast",
```

---

Maybe they're saying they're a web development enthusiast?